src/Codeception/Module/REST.php

```diff
@@ -1219,6 +1219,34 @@ public function dontSeeXmlResponseIncludes($xml)
     }
 
     /**
+     * Checks if the hash of a binary response is exactly the same as provided.
```

Comment:
a code example would be very helpful here. You can take one from your tests and add here.